Doc/library/sqlite3.rst

```diff
@@ -1666,6 +1611,85 @@ the context manager is a no-op.
 .. literalinclude:: ../includes/sqlite3/ctx_manager.py
 
 
+.. _sqlite3-uri-tricks:
+
+Working with SQLite URIs
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Some useful URI tricks include:
+
+* Open a database in read-only mode::
+
+    con = sqlite3.connect("file:template.db?mode=ro", uri=True)
+
+* Do not implicitly create a new database file if it does not already exist;
+  will raise :exc:`~sqlite3.OperationalError` if unable to create a new file::
+
+    con = sqlite3.connect("file:nosuchdb.db?mode=rw", uri=True)
+
+* Create a shared named in-memory database::
+
+    con1 = sqlite3.connect("file:mem1?mode=memory&cache=shared", uri=True)
+    con2 = sqlite3.connect("file:mem1?mode=memory&cache=shared", uri=True)
+    con1.execute("create table t(t)")
+    con1.execute("insert into t values(28)")
+    con1.commit()
+    rows = con2.execute("select * from t").fetchall()
+
+More information about this feature, including a list of parameters,
+can be found in the `SQLite URI documentation`_.
+
+.. _SQLite URI documentation: https://www.sqlite.org/uri.html
+
+
+.. _sqlite3-explanation:
+
+Explanation
+-----------
+
+.. _sqlite3-controlling-transactions:
+
+Transaction control
+^^^^^^^^^^^^^^^^^^^
+
+The ``sqlite3`` module does not adhere to the transaction handling recommended
+by :pep:`249`.
+
+If the connection attribute :attr:`~Connection.isolation_level`
+is not :const:`None`,
```

Comment:
Not your fault since you are just moving an existing, but for `None`, ` ``None`` ` is preferred.  This might belong in a different PR though, assuming you think it's worth fixing it.

Reply:
To note, I don't have a strong opinion, but ``:const:`None` `` is the general convention that @erlend-aasland has used so far through the revamped ``sqlite3`` docs, so it might be better to make this change separately if so. Is there somewhere this preference is documented?

Reply:
Oh, I was not aware of that convention; I've been using the reference version consistently in all my PRs. Is that also the convention for True and False?

Reply:
Changing all those in sqlite3.rst would be a large diff :) _If_ I were to do that, I prefer to do it in a separate PR.

Reply:
I'm going to mark this as resolved, as I won't be making any changes in this PR. I'll wait a little bit, in case other comments appear, though.

Reply:
Seems like its discussed in the [devguide](https://devguide.python.org/documentation/markup/#rest-inline-markup), but under the "reStructuredText Markup" page rather than the style guide, which mostly repeats general details of reST/Sphinx, and under the ``keyword`` role rather than the ``const`` role in which they are actually indexed under

![image](https://user-images.githubusercontent.com/17051931/181997304-0ec729f5-9dbf-4aed-b3d8-1b7971753eec.png)

Reply:
Ok, makes sense. Thanks for finding this.

Reply:
We should probably move that to the style guide.

Reply:
I consider it low priority, though :)

Reply:
> Is that also the convention for True and False?

See also https://github.com/python/devguide/pull/916, where I tried to make this more explicit.